src/components/frame/useFrame.ts

```diff
@@ -1,35 +1,58 @@
 import { RefObject, useEffect, useState } from "react";
-import connectToParent from "penpal/lib/connectToParent";
-import connectToChild from "penpal/lib/connectToChild";
+import { connectToParent, connectToChild } from "penpal";
+// eslint-disable-next-line
+// @ts-ignore
```

Comment:
what's the issue here ?

Reply:
ok fixed by updating the type declaration to the alias one